OpenRA.Mods.Common/Traits/Render/WithGateSpriteBody.cs

```diff
@@ -26,6 +26,9 @@ class WithGateSpriteBodyInfo : WithSpriteBodyInfo, IWallConnectorInfo, Requires<
 		[Desc("Wall type for connections")]
 		public readonly string Type = "wall";
 
+		[Desc("Override sequence to use when fully open (if non-null).")]
```

Comment:
The `(if non-null)` isn't obvious when looking at just the traits documentation. Just skip it.